/detect_node:
  ros__parameters:
    image_topic: "/image_raw/compressed"
    model_params:
      rknn_model: "yolov8.rknn"
      label: "labels.txt"
      img_size: 640
      conf: 0.5 # Confidence threshold
      iou: 0.8 # IOU threshold
      fps: 10 # fps limit
      enable: True

/visualize_node:
  ros__parameters:
    image_topic: "/image_raw/compressed"
    detections_topic: "/detections"
    preview: True
